bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -0,0 +1,423 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/about-base.html" %}
+
+{% block page_title %}{{ _('Mozilla Leadership') }}{% endblock %}
+{% block body_id %}about-leadership{% endblock %}
+{% block body_class %}sand about-leadership{% endblock %}
+
+{% block article %}
+
+  <h1 class="title-shadow-box">{{ _('Leadership') }}</h1>
+
+  <p>
+  {{ _('Our leadership team is made up of both paid staff and volunteers who represent a breadth of experience and expertise in building products that people love and empowering communities to make a significant impact on the project and in the world.') }}
+  {{ _('While these teams have different day-to-day responsibilities and areas of focus, they are all guided by our non-profit, open-source, community-based mission, identity and values.') }}
+  </p>
+
+  <nav id="page-nav">
+    <ul>
+      <li><a href="#management">{{ _('Management Teams') }}</a></li>
+      <li><a href="#reps-council">{{ _('Mozilla Reps Council') }}</a></li>
+      <li><a href="#directors">{{ _('Boards of Directors') }}</a></li>
+    </ul>
+  </nav>
+
+  <section id="management" class="brass">
+    <h2>{{ _('Management Teams') }}</h2>
+
+    <h3 id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{ _('Mozilla Corporation') }}</a></h3>
```

Comment:
(maybe 5) http://schema.org/Organization and the `member` properties for each person can be used here and there. Not a release blocker though.